tools/tu_collector/tests/tu_collector_test.py

```diff
@@ -60,3 +60,36 @@ def test_file_existence(self):
         self.assertTrue(
             any([path.endswith(os.path.join('/', 'hello.c')) for path in files]))
         self.assertIn('compilation_database.json', files)
+
+    def test_ctu_collection(self):
+        ctu_deps_dir = tempfile.mkdtemp()
```

Comment:
I prefer to create temporary directory on the following way:
https://github.com/Ericsson/codechecker/blob/ed84de269ce575a23252175ffb0461757b747c45/web/tests/functional/storage_of_analysis_statistics/test_storage_of_analysis_statistics.py#L163
If something happens between the creation and remove phase it will not mess up my system with unecessary directories.